src/content/docs/release-notes/mobile-release-notes/ios-release-notes/xcframework-agent-749

```diff
@@ -0,0 +1,22 @@
+---
+subject: XCFramework agent
+releaseDate: '2024-02-28'
+version: 7.4.9
+downloadLink: 'https://download.newrelic.com/ios_agent/NewRelic_XCFramework_Agent_7.4.9.zip'
+---
+
+## Fixed in this release
+
+* Fixed events maximum buffer size not maintaining after backgrounding.
+* Fixed issue when /connect has a missing harvest configuration.
+
+
+### Other notes
+
+* Added the ability to store harvests that previously would be lost if the application doesn't have internet connection.
+* Use the `+ (void) setMaxOfflineStorageSize:(unsigned int)megaBytes;` api to set the maximum amount of data that can be stored locally. [More info](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-offline-storage)
```

Comment:
```suggestion
* Use the `+ (void) setMaxOfflineStorageSize:(unsigned int)megaBytes;` API to set the maximum amount of data that can be stored locally. [More info](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-offline-storage)
```

Reply:
@mbruin-NR This doc link doesn't work. Does this page need to be created?

Reply:
https://github.com/newrelic/docs-website/pull/15875 
I didn't see that this pr wasn't merged because it had some issue. I think it's ready now.